onchain/rollups/test/foundry/consensus/authority/Authority.t.sol

```diff
@@ -108,6 +108,7 @@ contract AuthorityTest is TestBase {
         address _newConsensus
     ) public isMockable(address(_history)) {
         vm.assume(_owner != address(0));
+        vm.assume(_owner != msg.sender);
```

Comment:
The change introduced was to avoid when msg.sender == _owner, in this case we would find that all the calls performed by the tests where called by the same address. 
For example, by calling `authority.migrateHistoryToConsensus(_newConsensus);` as the caller is `msg.sender`, it'll not fail as expected if in a random scenario matches `_owner`.

Reply:
the caller is not `msg.sender`, but `this`

Reply:
Oh, my bad!

Reply:
I'll address this problem after PR #54 is merged.

Reply:
Addressed in [`f63787d`](https://github.com/cartesi/rollups/pull/49/commits/f63787da885a80f0e1ddc9b06099a0a1663c0939)